src/reporter/health.py

```diff
@@ -75,12 +80,15 @@ def get_health(with_geocoder=False):
         'status': 'pass'
     }
 
-    # Check crateDB (critical)
+    # Check defaultDB (critical)
+    db = default_backend().lower()
     try:
-        res = _check_critical(check_crate(), res, 'crateDB')
+        res = _check_critical(check_db(db), res, db)
```

Comment:
this change is already in #449, so perhaps we should merge #449 first and then double check what happens when rebasing, i.e. make sure nothing gets undone!